pom.xml

```diff
@@ -205,12 +205,12 @@
       <dependency>
         <groupId>org.powermock</groupId>
         <artifactId>powermock-module-junit4</artifactId>
-        <version>2.0.0-RC.4</version>
+        <version>2.0.0</version>
```

Comment:
it was a complete shock to me that this was even included - personally I feel neither mocito nor powermock should be in here and it is up to the individual plugins to use them if they desire.

for example if a new plugin includes a test project from a plugin built with older parent the dependency will automatically be shunted and is liable o break things.

/shrug

Reply:
Yes, it was deliberate since we have got a lot of fun while testing with Java 11

Reply:
> Also, if any plugin wants a different version, they can put a version to use and this will be ignored

for an individual plugin yes, but if another plugin depends on that plugin no, the dependency gets overridden